app/assets/stylesheets/application.css

```diff
@@ -15,3 +15,92 @@
  *= require foundation_and_overrides
 
  */
+
+
+@import url(http://fonts.googleapis.com/css?family=Pacifico);
+@import url(http://fonts.googleapis.com/css?family=Lato:300,400);
+
+nav.top-bar {
+  background:rgba(0,0,0,0.5);
+  background: #FFF;
+}
+
+.top-bar ul li h1 a {
+  font-family: Pacifico;
+  font-size: 1.4em;
+  color: #ff69b4 !important;
+}
+
+.top-bar .right li {
+  font-family: Pacifico;
+  font-size: 1em;
+  display: inline-block;
+  padding-right: 15px;
+  padding-top: 10px;
+}
+
+.top-bar .right li a {
+ color: #ff69b4 !important;
+}
+
+nav.top-bar ul.right li a:hover {
+  background-color: none !important;
+  background: none !important;
```

Comment:
Fixed most of them except for one, which is specific to the top nav bar hyperlink.